packages/manfred2html/src/engine/cv-x-wing-squadron/html-parts/language-section/labels/language-english-labels.const.ts

```diff
@@ -0,0 +1,10 @@
+import { LanguageLabels } from './language-label.model';
+
+export const englishLanguageLabels: LanguageLabels = {
```

Comment:
We only have four levels:

- Level one: Native/Bilingual
- level two: Fluent
- Level three: Intermediate
- Level four: Basic (Elementary proficiency y limited working proficiency)